qutebrowser/html/back.html

```diff
@@ -0,0 +1,26 @@
+{% extends "base.html" %}
+
+{% block script %}
+window.onload = function() {
+	var title = 'Suspended: ' + document.location.hash.substr(1);
+	var node = document.getElementsByTagName('h1')[0];
+	node.innerText = document.title = title;
+};
+setTimeout(function() {
+	/* drop first focus event, to avoid problems
+	(allow to go easily to newer history entries) */
+	var triggered = false;
+	window.onfocus = function() {
+		if (! triggered) {
+			triggered = true;
+			window.history.back();
+		}
+	};
+}, 1000);
```

Comment:
Why only after a second?

Reply:
Well, let's say a1...an n>3 is your real history and ai i \in [1, n-1] is your active entry.  
In this case qute://back will be inserted after ai.  
Your active page isn't the last page of the tab history as 1 <= i < n.  
If you want to move to the entry after ai you need to be fast than `window.history.back()`,  
that's why there's a delay.  
1sec delay -> you need to press LL within one second  
On the otherhand there's no way to restore a tab during the delay.  
So I decided to use one second as the delay should be over after selecting a tab via gt and also it should be possible to press LL within this time.